packages/manager/apps/container/src/container/legacy/server-sidebar/universe/TelecomSidebar.tsx

```diff
@@ -108,6 +108,11 @@ export default function TelecomSidebar() {
                           </span>
                         ),
                       }));
+                      // Set label to service name if is empty
+                      servicesLoaded.forEach((service) => {
+                        service.label = service.label ? service.label : service.serviceName;
+                      });
```

Comment:
This can be done in the previous mapping to avoid looping a second time on the services array
```suggestion
                      return services.map((service) => ({
                        ...service,
                        label: service.label || service.serviceName
                        keywords: service.extraParams?.length
                          ? service.extraParams[0]
                          : '',
                        icon: service.extraParams?.length && (
                          <span className={style.telecomLabel}>
                            {capitalize(service.extraParams[0])}
                          </span>
                        ),
                      }));
```